hbw/app/assets/stylesheets/hbw.sass

```diff
@@ -17,6 +17,9 @@
 
   @import 'hbw/branding'
 
+  body
+    font-size: $font-size
```

Comment:
I think we should do it only once: https://github.com/latera/homs/pull/480/files#diff-44af70099daac6ae6847f1ac32d66ab6R4